tests/suite/test_transport_server_tcp_load_balance.py

```diff
@@ -0,0 +1,88 @@
+import pytest
```

Comment:
there is a number of unused imports 

@lucacome @soneillf5 @vepatel @ciarams87 
perhaps it make sense to enable linting for python code in the pipeline?

Reply:
removed unused imports

Reply:
@pleshakov @lucacome @soneillf5 recomment black and flake8 for python linting:

`~/nginx/kubernetes-ingress/tests ingress-mtls-tests*
tests ❯ flake8 suite/test_ingress_mtls.py
suite/test_ingress_mtls.py:39: [F541] f-string is missing placeholders
suite/test_ingress_mtls.py:40: [F541] f-string is missing placeholders
suite/test_ingress_mtls.py:53: [F541] f-string is missing placeholders
suite/test_ingress_mtls.py:56: [F541] f-string is missing placeholders
suite/test_ingress_mtls.py:59: [F541] f-string is missing placeholders
suite/test_ingress_mtls.py:146: [F541] f-string is missing placeholders
suite/test_ingress_mtls.py:211: [F541] f-string is missing placeholders

~/nginx/kubernetes-ingress/tests ingress-mtls-tests*
tests ❯ black suite/test_ingress_mtls.py -l 100      
All done! ✨ 🍰 ✨
1 file left unchanged.
`